models/stg_asana__project.sql

```diff
@@ -25,8 +25,6 @@ final as (
         id as project_id,
         archived as is_archived,
         cast(created_at as {{ dbt_utils.type_timestamp() }}) as created_at,
-        current_status, 
-        cast(due_date as {{ dbt_utils.type_timestamp() }}) as due_date,
```

Comment:
Why are we removing `current_status` and `due_date`? They don't seem to be listed as depreciated fields?

Reply:
Hi @fivetran-joemarkiewicz, working through this now, the docs for the Project schema lists these columns as deprecated:

https://developers.asana.com/docs/project - it instructs to use the status_update resource and due_on resource accordingly.

Unless I'm misinterpreting the docs?

Reply:
Thanks for sharing @fivetran-poonamagate! I am curious however that we did not highlight these fields as being depreciated in the Fivetran connector docs 🤔

Would you be able to confirm with the Product manager (I believe it is Erin) if these fields are depreciated within the connector.

Reply:
Hi @fivetran-joemarkiewicz I created an Asana connector to test my changes to the package and it brought in the fields that are listed as deprecated on Asana's docs. Going forward, shall I keep these fields I am deleting and ensure that my changes to the package align to our internal docs rather than the connector specific docs? That was perhaps an over-sight on my end, my bad!

Reply:
@fivetran-poonamagate I would keep those in for now, but open an issue with our connector team to take a closer look. I would want to understand if our connector is maybe recreating these fields somehow. 🤔